custom/opm/reports.py

```diff
@@ -500,7 +500,7 @@ def join_rows(self, rows):
         if len(rows) == 1:
             return rows[0]
         def zip_fn((i, values)):
-            if i == self.column_index('num_children'):
+            if i == self.column_index('num_children') or i == self.column_index('year_end_bonus_cash'):
```

Comment:
@czue 
In report we combine children cases for one mother and we sum all ```int``` values. For example: when we have mother with two children then in each we have birth spacing bonus (2000). When we sum this values we get 4000.

You can notice how we combine rows in report when you add debug option in url:
https://www.commcarehq.org/a/opm/reports/custom/beneficiary_payment_report/?hierarchy_block=Wazirganj&hierarchy_gp=Kari&hierarchy_awc=Bhura+-+(43)&month=02&year=2016&is_open=open&debug=true